api_bonus.yaml

```diff
@@ -14,34 +14,8 @@ paths:
       operationId: startBonusEligibilityCheck
       summary: Start bonus eligibility check (ISEE)
       responses:
-        "201":
-          description: Request created.
-          schema:
-            $ref: "#/definitions/InstanceId"
-          headers:
-            Location:
-              type: string
-              description: |-
-                Location (URL) of created request resource.
-                A GET request to this URL returns the request status and details.
-        "202":
-          description: Processing request.
-        "401":
```

Comment:
this API could return 401 with session_token evaluation (bearerSessionTokenAuth) or 500 if something goes wrong with redis client

Reply:
You're right! My bad.

Question: should we evaluate the token as well? Shouldn't we just return "dismissed"?

Reply:
The API definition requires an Authorization header. If the token is invalid maybe is better returns the proper error evaluating it.